### PR TITLE
Added: docs and home page  and avatar feature

### PR DESCRIPTION
**Pull request by sushil-mod**

### CODE REVIEW

File: Components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.avatar-lg{
+    width: 6rem;
+    height: 6rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
+
```
Comment (harshitadaswani) [inline Components/avatar.css]: remove unnecessary space

File: Components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.avatar-lg{
+    width: 6rem;
+    height: 6rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
```
Comment (harshitadaswani) [inline Components/avatar.css]: combine common properties

File: Components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+.avatar-img{
+    width: 100%;
+    height: 100%;
+}
+
+
+.avatar-text{
+
```
Comment (harshitadaswani) [inline Components/avatar.css]: remove space

File: Documentation/document.css [hunk 4]
```diff
@@ -43,14 +45,34 @@ aside{
 }
 
 main{
-    color: white;
-    background-color: turquoise;
```
Comment (harshitadaswani) [inline Documentation/document.css:47]: use color variables

File: Documentation/document.css [hunk 5]
```diff
@@ -43,14 +45,34 @@ aside{
...
-    flex-grow: 1;
+   
+    width: 100%;
+    height: auto;
+    /* color: rgb(134, 190, 223); */
```
Comment (harshitadaswani) [inline Documentation/document.css]: remove comment

File: Documentation/document.css [hunk 6]
```diff
@@ -43,14 +45,34 @@ aside{
...
     margin-bottom: 20px;
-}
\ No newline at end of file
+} */
+
+
+.doc-header{
+    padding-left: 2rem;
+    font-size: 2.5rem;
+    color: rgb(134, 190, 223);
```
Comment (harshitadaswani) [inline Documentation/document.css]: use color variable

File: Documentation/document.css [hunk 7]
```diff
@@ -43,14 +45,34 @@ aside{
...
+} */
+
+
+.doc-header{
+    padding-left: 2rem;
+    font-size: 2.5rem;
+    color: rgb(134, 190, 223); 
+}
+.doc-avatar{
+    /* border: 2px solid white; */
```
Comment (harshitadaswani) [inline Documentation/document.css]: remove comment

File: Documentation/document.css [hunk 8]
```diff
@@ -43,14 +45,34 @@ aside{
...
+    display: flex;
+    flex-direction: column;
+    margin-left: 2rem;
+}
+.doc-avatar p {
+    font-size: 1.5rem;
+    padding-bottom: 2rem;
+    color: rgb(134, 190, 223);
+    /* text-align: center;  */
```
Comment (harshitadaswani) [inline Documentation/document.css]: remove comment

File: Documentation/document.html [hunk 9]
```diff
@@ -21,27 +22,63 @@
...
-            <div class="content"><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus minus sequi incidunt officiis aliquid totam, dolore mollitia, rerum voluptatem maxime eius quaerat ipsa fugit? Assumenda necessitatibus deserunt reiciendis itaque alias.</p> </div>
-            <div class="content"><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus minus sequi incidunt officiis aliquid totam, dolore mollitia, rerum voluptatem maxime eius quaerat ipsa fugit? Assumenda necessitatibus deserunt reiciendis itaque alias.</p> </div>
-            <div class="content"><p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus minus sequi incidunt officiis aliquid totam, dolore mollitia, rerum voluptatem maxime eius quaerat ipsa fugit? Assumenda necessitatibus deserunt reiciendis itaque alias.</p> </div>
-                
+               <iframe src="https://www.thiscodeworks.com/embed/6206b7583d90130015398b94" style="width: 100%; height: 151px;" frameborder="0"></iframe>
+
+               <p>Avatar with diffirent size</p>
```
Comment (harshitadaswani) [inline Documentation/document.html]: typo: different

File: style.css [hunk 10]
```diff
@@ -17,7 +17,7 @@ h1{
 
 }
 
-header{
+.nav-header{
     display: flex;
```
Comment (harshitadaswani) [inline style.css:21]: remove space

**Review by harshitadaswani — APPROVED** [review]
Refactor the suggested changes and you are good to go

**Review by Raghav888 — APPROVED** [review]

File: Components/avatar.css (hunk 1 above)
Comment (sushil-mod) [reply Components/avatar.css]: space removed in avatar.css

File: Components/avatar.css (hunk 2 above)
Comment (sushil-mod) [reply Components/avatar.css]: done ,combined border radius and overflow into avatar-round done

File: Components/avatar.css (hunk 3 above)
Comment (sushil-mod) [reply Components/avatar.css]: space removed in avatar.css

File: Documentation/document.css (hunk 5 above)
Comment (sushil-mod) [reply Documentation/document.css]: done ,comment removed ffrom document css

File: Documentation/document.css (hunk 6 above)
Comment (sushil-mod) [reply Documentation/document.css]: done ,doc-txt-color variable used instead of color

File: Documentation/document.css (hunk 7 above)
Comment (sushil-mod) [reply Documentation/document.css]: done , removed comment

File: Documentation/document.css (hunk 4 above)
Comment (sushil-mod) [reply Documentation/document.css:47]: done

File: Documentation/document.css (hunk 8 above)
Comment (sushil-mod) [reply Documentation/document.css]: comment removed

File: Documentation/document.html (hunk 9 above)
Comment (sushil-mod) [reply Documentation/document.html]: changed to  different

File: style.css (hunk 10 above)
Comment (sushil-mod) [reply style.css:21]: removed extra space from style css and also used color variables

**Pull request merged**